bugsnag/django/__init__.py

```diff
@@ -45,14 +46,23 @@ def add_django_request_to_notification(notification):
 
     if getattr(request, "session", None):
         notification.add_tab("session", dict(request.session))
-    notification.add_tab("request", {
+    req = {
         'method': request.method,
         'path': request.path,
         'encoding': request.encoding,
         'GET': dict(request.GET),
         'POST': dict(request.POST),
         'url': request.build_absolute_uri(),
-    })
+    }
+    if 'CONTENT_TYPE' in request.META:
+        if request.META['CONTENT_TYPE'] == 'application/json':
+            req["content_type"] = request.META['CONTENT_TYPE']
+            if req["method"] == "POST":
```

Comment:
This block probably needs to be wrapped in a `try`, so we don't crash if the body doesn't decode or the JSON fails to parse

I think you can test this by sending some nonsense JSON and checking that POST is empty. Without a `try` here, the whole request tab will empty if this fails

Reply:
Fixed in 4b59d6cf5ee86e11ede693f10421c1f95379d044